code/game/mecha/mech_bay.dm

```diff
@@ -1,84 +1,26 @@
-/turf/simulated/floor/mech_bay_recharge_floor
+/turf/simulated/floor/mech_bay_recharge_floor //I'll remove this in a future PR, as it requires map changes.

```

Comment:
Don't add in comments like this, and there's nothing wrong with this turf, specifically, anyway.